dotCMS/src/main/java/com/dotcms/util/VoidDelegate.java

```diff
@@ -0,0 +1,38 @@
+package com.dotcms.util;
+
+
+import com.dotmarketing.exception.DotDataException;
+
+/**
+ * <p>
+ *     A delegate class is basically a class that executes a specific task, it is usually used to decouple some generic task from the specific one (the delegate will be the specialization in the routine).
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 204 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)